adagrams/game.py

```diff
@@ -1,11 +1,118 @@
+import random
+
+LETTER_POOL = {"A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, 
```

Comment:
Great job making these constant global variables. This makes the functions less cluttered and lets the variables be accessible to other functions.